pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -53,17 +56,32 @@
  * If the input is single value, the output type will be input type. Otherwise, the output type will be double.
  */
 public class AggregateOperator extends BaseOperator<TransferableBlock> {
+
+  interface Merger extends BiFunction<Object, Object, Object> {
+  }
```

Comment:
nit: move next to Accumulator? also make it private? i don't foresee anyone uses this interface and accumulate is already a private member class

Reply:
it used in testing, so it's package private :) I can move it